atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -169,6 +169,16 @@ public abstract class TransactionManagers {
 
     abstract Set<Schema> schemas();
 
+    /**
+     * @deprecated This method is not used and should not be overriden from the default. It only exists because deleting
+     * it would be an ABI break.
+     */
+    @Value.Default
+    @Deprecated
```

Comment:
Might not want to deprecate this since it will block automatic upgrades in any repos that have `-Werror` and aren't actively maintained.